### PR TITLE
acceptance tests: update Jira version to current LTS release, 8.13.4

### PR DESCRIPTION
**Pull request by kenyon**
Update Jira version to current LTS release, 8.13.4.

Needs #340 for tests to pass.

### CODE REVIEW

File: spec/acceptance/mysql_spec.rb [hunk 1]
```diff
@@ -92,11 +92,11 @@ class { 'jira::facts': }
   end
 
   describe command('wget -q --tries=24 --retry-connrefused --no-check-certificate --read-timeout=10 -O- localhost:8081') do
-    its(:stdout) { is_expected.to match(%r{8\.9\.0}) }
+    its(:stdout) { is_expected.to match(Regexp.new(Regexp.escape('8.13.3'))) }
```
Comment (ekohl) [inline spec/acceptance/mysql_spec.rb:95]:
```suggestion
    its(:stdout) { is_expected.to include('8.13.3') }
```

File: spec/acceptance/mysql_spec.rb [hunk 2]
```diff
@@ -92,11 +92,11 @@ class { 'jira::facts': }
...
   end
 
   describe command('wget -q --tries=24 --retry-connrefused --no-check-certificate --read-timeout=10 -O- https://localhost:8443') do
-    its(:stdout) { is_expected.to match(%r{8\.9\.0}) }
+    its(:stdout) { is_expected.to match(Regexp.new(Regexp.escape('8.13.3'))) }
```
Comment (ekohl) [inline spec/acceptance/mysql_spec.rb:99]:
```suggestion
    its(:stdout) { is_expected.to include('8.13.3') }
```

**Review by ekohl — COMMENTED** [review]
I think these also work. Other than that :+1: for using a current version.

File: spec/acceptance/default_parameters_spec.rb [hunk 3]
```diff
@@ -67,7 +67,7 @@ class { 'jira::facts': }
   end
 
   describe command('wget -q --tries=24 --retry-connrefused --read-timeout=10 -O- localhost:8080') do
-    its(:stdout) { is_expected.to match(%r{7\.13\.0}) }
+    its(:stdout) { is_expected.to match(Regexp.new(Regexp.escape('8.13.3'))) }
```
Comment (ekohl) [inline spec/acceptance/default_parameters_spec.rb:70]:
```suggestion
    its(:stdout) { is_expected.to include('8.13.3') }
```

**Review by ekohl — COMMENTED** [review]
Any reason not to update the version in `init.pp` and not specify it in the acceptance tests?

**Comment by kenyon** [comment]
Closing in favor of 496b9d8 in #334.

**Pull request closed without merging**